securesystemslib/interface.py

```diff
@@ -516,13 +516,15 @@ def generate_and_write_ed25519_keypair(filepath=None, password=None):
   # to final destination.
   file_object = tempfile.TemporaryFile()
 
-  # Generate the ed25519 public key file contents in metadata format (i.e.,
-  # does not include the keyid portion).
```

Comment:
Do you know why we never included the keyid here? It doesn't make sense for me, because if we don't provide a filepath we are even naming the keypair files <KEYID>.{pub,key}